source-code/core/src/lint/linter.ts

```diff
@@ -3,6 +3,7 @@ import type * as ast from "@inlang/core/ast"
 import type { LintedResource, LintRule, Visitors } from "./rule.js"
 import { createReportFunction } from "./report.js"
 import type { Language } from "@inlang/core/ast"
+import structuredClone from "@ungap/structured-clone"
```

Comment:
@felixhaeberle ALARM WHAT IS THIS???

@felixhaeberle polyfill in the ide extension. Not in core! Please change this.

Reply:
There was a misunderstanding with polyfilling which module. Fixed that with the  latest commit.